csharp-advanced-topics/StaticClasses/StaticClassesTest/StaticClassUnitTest.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using StaticClasses;
+
+namespace StaticClassesTest
+{
+    [TestClass]
+    public class StaticClassUnitTest
+    {
+        [TestMethod]
+        public void GivenStudentDate_ThenReturnCorrectAge()
+        {
+            //assign variables to the Student class
+            Program.Student.Id = 1;
+            Program.Student.Name = "John Doe";
+            Program.Student.DateOfBirth = new DateTime(1994, 12, 31);
+            int expectedAge = 26;
+            int actualAge = Program.Student.CalculateAge(Program.Student.DateOfBirth);
+
+            Assert.AreEqual(expectedAge, actualAge);
+        }
+
+        [TestMethod]
+        //test the non-static version of the same class
+        public void GivenStudentAgeNonStatic_ThenReturnCorrectAge() 
+        {
+            //instantiate non static class
+            CollegeStudent student = new CollegeStudent();
```

Comment:
You can improve this by using var, or new(). We prefer using var, but it's up to you.